proto/cosmos/auth/v1beta1/auth.proto

```diff
@@ -45,3 +45,11 @@ message Params {
   uint64 sig_verify_cost_ed25519   = 4 [(gogoproto.customname) = "SigVerifyCostED25519"];
   uint64 sig_verify_cost_secp256k1 = 5 [(gogoproto.customname) = "SigVerifyCostSecp256k1"];
 }
+
+message PubKeyMapping {
```

Comment:
Do we really need this?

Reply:
The address field is needed to generate keys at init genesis.

Reply:
I think that's not required, unless we need account pubkeys history (all updates data). wdyt